frameit/frames_generator/Rakefile

```diff
@@ -79,4 +100,36 @@ def convert_image_resource(path)
   image.write(path)
 end
 
+def measure_slot(path)
+  tmp_output = Time.now.to_i.to_s + ".png"
+
+  convert = MiniMagick::Tool::Convert.new
```

Comment:
Maybe add a few comment on the individual parts of the command what they do (similar to other PR)?